INFO     run_all_case.py:test_RegisterCase.py:30 当前用例名称：密码输入17位
INFO     run_all_case.py:test_RegisterCase.py:31 当前测试用例请求参数：{'password': '12345678901234567', 'phone': '18804320002', 'v_code': '123456'}
INFO     run_all_case.py:test_RegisterCase.py:32 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
INFO     run_all_case.py:test_RegisterCase.py:33 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '验证码无效，请重新获取', 'debug_error': ''}